infra/aws/terraform/modules/eks-prow-iam/policy_apply.tf

```diff
@@ -60,6 +60,7 @@ data "aws_iam_policy_document" "eks_apply" {
       "ec2:ModifyInstanceAttribute",
       "ec2:TerminateInstances",
       "ec2:ImportKeyPair",
+      "eks:CreateAccessEntry",
```

Comment:
fyi - there is also an `eks:UpdateAccessEntry`

Reply:
Added, thanks!